python/foglamp/services/south/server.py

```diff
@@ -180,15 +180,18 @@ def __init__(self):
                 # pollInterval is expressed in milliseconds
                 sleep_seconds = int(self._plugin_handle['pollInterval']['value']) / 1000.0
                 await asyncio.sleep(sleep_seconds)
-                # If successful, then set retry count back to 1, meaning that
-                # only in case of 3 successive failures, exit.
-                try_count = 1
             except KeyError as ex:
+                try_count = 2
                 _LOGGER.exception('Key error plugin {} : {}'.format(self._name, str(ex)))
+            except exceptions.QuietError:
+                try_count = 2
+                await asyncio.sleep(_TIME_TO_WAIT_BEFORE_RETRY)
             except (Exception, RuntimeError, exceptions.DataRetrievalError) as ex:
-                try_count += 1
-                _LOGGER.exception('Failed to poll for plugin {}, retry count: {}'.format(self._name, try_count))
+                try_count = 2
+                _LOGGER.error('Failed to poll for plugin {}'.format(self._name))
+                _LOGGER.debug('Exception poll plugin {}'.format(str(ex)))
```

Comment:
only error ?! and actually exceptions

Reply:
Error hides the stack trace. Since this error may come in plenty, we don't need these verbose errors each time and crowd the syslog.

Reply:
To avoid crowding of syslog, detailed error from plugin vide debug mode only .